syntax = "proto3";

package helloexample;

message HelloRequest {
  string name = 1;
}

message HelloReply {
    string message = 1;
}

// Interface exported by the server.
service Greeter {
    rpc SayHello(HelloRequest) returns (HelloReply) {}
    rpc SayHelloAgain(HelloRequest) returns (HelloReply) {}
}
